app/views/jobs/index.html.erb

```diff
@@ -1,19 +1,38 @@
 <h1> Jobs!</h1>
 
-
 <header style="text-align: right">
-  <%=  button_to "New Job", new_job_path, method: :get %>
+  <%= button_to "New Job", new_job_path, method: :get %>
 </header>
 
-<% @jobs.each do |job| %>
-  <div class="job">
-    <div class="container">
-      <h4><b><%= job.title%></b></h4>
-      <p><%= job.link%></p>
-      <p><%= job.company%></p>
-      <p><%= job.location%></p>
-      <p><%= button_to "Edit", edit_job_path(job.id), method: :get %></p>
-      <p><%= button_to "Delete", job, method: :delete, data: {turbo_confirm: "Are you sure?"} %></p>
-    </div>
-  </div>
-<% end %>
\ No newline at end of file
+<h2> Interested</h2>
+<% @jobs.select{|job| job.has_status?("Interested")}.each do |job| %>
+  <%= render partial: "job_card_partial", locals: { job: job } %>
+<% end %>
+
+<h2> Applied</h2>
+<% @jobs.select{|job| job.has_status?("Applied")}.each do |job| %>
+  <%= render partial: "job_card_partial", locals: { job: job } %>
+<% end %>
+
+
+<h2> Interviewed</h2>
+<% @jobs.select{|job| job.has_status?("Interviewed")}.each do |job| %>
+  <%= render partial: "job_card_partial", locals: { job: job } %>
+<% end %>
+
+<h2> Offered</h2>
+<% @jobs.select{|job| job.has_status?("Offered")}.each do |job| %>
+  <%= render partial: "job_card_partial", locals: { job: job } %>
+<% end %>
+
+
+<h2> Rejected</h2>
+<% @jobs.select{|job| job.has_status?("Rejected")}.each do |job| %>
+  <%= render partial: "job_card_partial", locals: { job: job } %>
+<% end %>
+
+
+<h2> No Status</h2>
```

Comment:
I don't think we should have a `No Status` option. All jobs should have a status.